templates/dashboard.mustache

```diff
@@ -169,6 +170,17 @@
                             {{#str}} dashboard:changesource, local_sitsgradepush {{/str}}
                             <span class="sr-only"> for {{mabseq}} {{mabname}}</span>
                         </a>
+                        <button type="button"
+                                class="btn btn-danger btn-sm {{#hideremovesourcebutton}}d-none{{/hideremovesourcebutton}}"
+                                data-confirmation="modal"
```

Comment:
Ha ha ha ha!  Amazing.

It's still using js to add dom elements and remove them (bad for performance), but its moodle core, so we don't have to write our own custom js (good for maintainability).

That doesn't mean i actually like it.

10/10 for finding this @aydevworks.

Genuinely deserves a golf clap :)